### DISCUSSION TITLE
What is Milestones?

### DISCUSSION THREAD

**Discussion by pr0xy-t** [Vector35/binaryninja-api#3779] [Q&A] [answered]
I know some issues are linked to [Milestones](https://github.com/Vector35/binaryninja-api/milestones).
I would like to know how each issue is categorized.

**Comment by pr0xy-t** [comment, marked as answer]
(I would like to know about description of each Milestone.)

**Reply by psifertex** [reply]
Milestones are upcoming releases. Sometimes they have a theme and sometimes they are simply a collection of fixes and features. 

They are alphabetical so you can know which is next and we plan to do approximately 3-4 per year but don't have hard dates otherwise. 

We had notional plans that Braize is for embedded and Coruscant is for mobile (arm, iOS, Android) but that is subject to change which is why they aren't currently documented that way. 

Let me know if that explains things well enough.

**Reply by pr0xy-t** [reply]
I see. So milestones are released in **alphabetical** order, and they are mainly a collection of features by type.
(The current milestone naming convention was changed from the previous one ( "3.1 (Performance)", "3.2 (Windows)", ... ) so I wasn't sure which milestone would be released next.)

If a  ultra-cool mobile feature is to be developed, will we have to wait a few months after "Braize" releases it because the  ultra-cool mobile feature will be linked to the "Coruscant" milestone?

**Reply by psifertex** [reply]
We don't hold back features if they're urgent or a developer works on it. For example, even though it was originally for the "embedded" release, we landed the [memory map in 3.2](https://binary.ninja/2022/10/28/3.2-released.html#segments-and-sections-editing-ui). That's one of the reasons we changed the names to not be so narrowly defined. It might give people the impression it _only_ contains features for that focus which definitely isn't the case.

**Reply by psifertex** [reply]
Note that I'm marking this top reply as the answer even though it's the whole thread that contains the answer. :-)
